airflow/contrib/operators/sagemaker_create_training_job_operator.py

```diff
@@ -62,16 +70,23 @@ def __init__(self,
                  sagemaker_conn_id=None,
                  job_name=None,
                  training_job_config=None,
+                 use_db_config=False,
+
                  *args, **kwargs):
         super(SageMakerCreateTrainingJobOperator, self).__init__(*args, **kwargs)
 
         self.sagemaker_conn_id = sagemaker_conn_id
         self.job_name = job_name
         self.training_job_config = training_job_config
+        self.use_db_config = use_db_config
 
     def execute(self, context):
+        if self.job_name != self.training_job_config['TrainingJobName']:
+            raise AirflowException("Job_name is not the the same as the one in "
+                                   "training_job_config")
```

Comment:
According to previous comment, if we don't need job_name, we don't need to check here.

Reply:
removed